Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Andy Russell <arussell123@gmail.com>"]
 
 [dependencies]
 env_logger = "0.3"
-mioco = "0.4.1"
+mioco = "*"
```

Comment:
I don't want to depend on a wildcard dep here, can you update to a specific version?